.circleci/config.yml

```diff
@@ -1273,7 +1273,7 @@ jobs:
       - run:
           name: Install runtime dependencies
           command: |
-            pacman --noconfirm -Syu --noprogressbar --needed base-devel boost cmake z3 cvc4 git openssh tar
+            pacman --noconfirm -Syu --noprogressbar --needed z3
```

Comment:
Installing even z3 on runtime only makes sense if we had installed it at build time and built with ``USE_Z3_DLOPEN`` (which would again break our tests on Z3 updates, I guess) - or what's the rationale of keeping it in the current configuration?